pkg/graph/resolvers/search_change_table.go

```diff
@@ -33,6 +33,7 @@ const (
 )
 
 // Embed the template files
+//
```

Comment:
Can you revert this? (This is the definition of a nit pick 😂)

Reply:
@ClayBenson94, this one is a weird one! I actually introduced it because the linter is complaining about it (even though I didn't touch the file 😂 . So I'd have to force the commit if I don't commit this change